examples/code-interpreter-agent-on-adk/README.md

```
@@ -50,8 +52,184 @@ The guide walks you through the process of creating a simple [ADK](https://googl
+}
+
+func (s *Sandbox) RunCommand(cmd string) (string, error) {
+     body, _ := json.Marshal(map[string]string{"command": cmd})
+     url := fmt.Sprintf("%s/sandboxes/%s/commands", s.client.baseURL, s.ID)
+     resp, err := s.client.httpClient.Post(url, "application/json", bytes.NewReader(body))
+     if err != nil {
+        return "", fmt.Errorf("run command: %w", err)
+     }
+     defer resp.Body.Close()
+
+     var result struct {
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)